meta-oe/recipes-connectivity/hiba/hiba_git.bb

```diff
@@ -0,0 +1,40 @@
+
+SUMMARY = "Google Host Identity Based Authorization"
+DESCRIPTION = "HIBA is a system built on top of regular OpenSSH \
+  certificate-based authentication that allows to manage flexible \
+  authorization of principals on pools of target hosts without the need to \
+  push customized authorized_users files periodically."
+
+S = "${WORKDIR}/git"
+
+LICENSE = "BSD"
+LIC_FILES_CHKSUM = "file://${S}/LICENSE;md5=7e161abe9a4585310725dd86c28d1ae2"
+
+SRC_URI = "git://github.com/google/hiba;protocol=https;branch=main"
+SRCREV = "3199239c6a0a89dbd8f155204f44798c78195ab0"
+
+PV = "1.0+git${SRCPV}"
+
+inherit autotools
+
+DEPENDS = "openssl openssh zlib"
+RDEPENDS:${PN} = "bash"
+
+def get_oeconf(d):
+  import glob
+  workdir = d.getVar("WORKDIR")
+  openssh_paths = glob.glob("{}/../../openssh/*/openssh-*".format(workdir))
+  if len(openssh_paths) != 1:
+    raise Exception("openssh_paths does not have only one path: {}".format(openssh_paths))
+  
+  return "--prefix=/usr/ --with-opensshdir={}".format(openssh_paths[0])
+
+EXTRA_OECONF = "${@get_oeconf(d)}"
+
+# Make sure that MACHINE_FEATURES includes hiba before continuing
+do_configure:prepend(){
+  if [[ "${MACHINE_FEATURES}" != *"hiba"* ]]; then
```

Comment:
No bashism please, we do support other shells e.g. dash for parsing generated task scripts

Reply:
Updated. Is this better? Or did I do it wrong again?